lib/index.js

```diff
@@ -130,10 +128,10 @@ Keen.prototype.identify = function(identify) {
  * @param {Track} track
  */
 
-Keen.prototype.track = function(track) {
+KeenSegment.prototype.track = function(track) {
   var props = track.properties();
   this.addons(props, track);
-  this.client.addEvent(track.event(), props);
+  this.client.recordEvent(track.event(), props);
```

Comment:
Any notable changes in regards to functionalities between these two functions? Or is it just an syntactical update?